documentation/explainers/dvm_interfaces.md

```diff
@@ -0,0 +1,167 @@
+# How Users and Voters Interact with the DVM
+
+This document explains how various participants interact with the DVM. There are three primary _interfaces_ by which
+different participants interact with the DVM:
+
+- The _Oracle Interface_ is used by financial contracts that need a price to request it and then retrieve it once it's
+available.
+
+- The _Store Interface_ is used by financial contracts to compute and pay the required DVM fees.
+
+- The _Voting Interface_ is used by voters to vote for prices to resolve pending price requests.
+
+## The Oracle Interface
+
+The Oracle Interface is used by financial contracts to retrieve prices. This interface can only be used by financial
+contracts that request prices _sparingly_. This is dependent on the specifics of the financial contract, but, in
+general, prices should only be requested to resolve disputes resolution and settlement.
```

Comment:
Maybe delete either "resolve" or "resolution" from this sentence?

Reply:
Yes! Thanks for finding this - I spotted it as I was leaving Friday, but couldn’t find it today.